### DISCUSSION TITLE
[Bug]: Image resizing error inside a canvas

### DISCUSSION THREAD

**Discussion by NicolaCocozza** [fabricjs/fabric.js#9149] [General]
I use the library in my angular project and have a problem resizing an image. In particular I have a canvas with dimensions of width 100 px and height 100 px and I perform a calculation to scale to insert a very large image in the canvas. The image in question has these properties width 5514 px and height 2500 px and for resolution width 300px and height 300px the factor si calculated scalability and about 0.018. The version that I use of the library is 4.4.0 while for angular I use version 10. Can anyone tell me how I could fix this bug? Thank you already in advance.

**Comment by ShaMan123** [comment]
Many angular bugs are being reported
And also it might be related to specific browsers
As always you should provide a clean repro (bare fabric, no frameworks) and then we all can see if it is on our side on not
